### PR TITLE
[Doc] PR for Update kubernetes config page

### PR DESCRIPTION
**Pull request by lizayugabyte**
Addresses https://yugabyte.atlassian.net/browse/PLAT-3283

Preview: https://deploy-preview-11690--infallible-bardeen-164bc9.netlify.app/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes/

### CODE REVIEW

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ Expect the following output:
 serviceaccount/yugabyte-platform-universe-management created
 ```
 
-You need to grant access to this ServiceAccount using ClusterRoles and Roles, as well as ClusterRoleBindings and RoleBindings, thus allowing it to manage the YugabyteDB universe's resources for you.
+You need to grant access to this service account using ClusterRoles and Roles, as well as ClusterRoleBindings and RoleBindings, thus allowing it to manage the YugabyteDB universe's resources for you.<br><br>
 Ensure that you have replaced the `namespace` from the commands with the correct namespace of the previously created ServiceAccount.
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md:99]: a bit confusing; maybe something like "Replace `namespace` in the commands that follow with the namespace of the service account you created."
And in the commands, is `<SA_NAMESPACE>` different from `<namespace>`?

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md [hunk 2]
```diff
@@ -95,7 +95,7 @@ Expect the following output:
...
 
 The tasks you can perform depend on your access level.
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md:101]: subheads might be nice - Global, Namespace admin, and Namespace restricted

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md [hunk 3]
```diff
@@ -166,7 +166,7 @@ kubectl auth can-i \
     {poddisruptionbudgets|services|statefulsets|secrets|pods|pvc}
 ```
 
-### `kubeconfig` file for a Kubernetes cluster
+### `kubeconfig` file
 
 You can create a `kubeconfig` file for previously created `yugabyte-platform-universe-management` service account as follows:
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md:171]: for the previously created

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md [hunk 4]
```diff
@@ -192,42 +192,37 @@ You can create a `kubeconfig` file for previously created `yugabyte-platform-uni
...
+Continue configuring your Kubernetes provider by clicking **Add region** and completing the **Add new region** dialog, as follows:
+
+- Use the **Region** field to select the region.
 
-- Use the `Region` field to select the region.
 - Use the **Zone** field to select a zone label that should match with your failure domain zone label `failure-domain.beta.kubernetes.io/zone`.
+
 - Optionally, use the **Storage Class** field to enter a comma-delimited value. If you do not specify this value, it would default to standard. You need to ensure that this storage class exists in your Kubernetes cluster.
+
 - Use the **Namespace** field to specify the namespace. If provided SA has the `Cluster Admin` permissions, you are not required to complete this field. The SA used in the provided `kubeconfig` file should have access to this namespace.
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md:221]: "SA" - service account? acronym not used previously

**Review by ddhodge — APPROVED** [review]
Minor comments

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md:99]: Rephrased.
sa_namespace is the one that was created previously. So, as far as I understand, if the service account's namespace was yb_sa, the command would be:

```
| sed "s/namespace: yb_sa/namespace: yb_sa"/g \
  | kubectl apply -n yb_sa -f -
```

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md (hunk 4 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md:221]: Fixed

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md (hunk 2 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md:101]: Added some formatting

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md [hunk 5]
```diff
@@ -131,13 +129,13 @@ kubectl auth can-i \
 
 ```sh
 curl -s https://raw.githubusercontent.com/yugabyte/charts/master/rbac/platform-namespaced-admin.yaml \
-  | sed "s/namespace: <SA_NAMESPACE>/namespace: <namespace>"/g \
+  | sed "s/namespace: <serviceaccount_namespace>/namespace: <namespace>"/g \
```
Comment (yogendra) [inline docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md:132]: `platform-namespaced-admin.yaml` has a placeholder <SA_NAMESPACE>. So this updated sed command does not replace the placeholder anymore.

Either

1. Change the command to use the correct placeholder
2. Change the `platform-namespaced-admin.yaml` to use the correct placeholder

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md [hunk 6]
```diff
@@ -95,30 +92,31 @@ Expect the following output:
...
   | kubectl apply -n <namespace> -f -
 ```
 
 **Global Restricted** can grant access to only the specific cluster roles to create and manage YugabyteDB universes across all the namespaces in a cluster using the following command:
 
 ```sh
 curl -s https://raw.githubusercontent.com/yugabyte/charts/master/rbac/platform-global.yaml \
-  | sed "s/namespace: <SA_NAMESPACE>/namespace: <namespace>"/g \
+  | sed "s/namespace: <serviceaccount_namespace>/namespace: <namespace>"/g \
```
Comment (yogendra) [inline docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md:113]: `platform-global.yaml` has a placeholder <SA_NAMESPACE>. So this updated sed command does not replace the placeholder anymore.

Either

1. Change the command to use the correct placeholder
2. Change the `platform-global.yaml` to use the correct placeholder

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md [hunk 7]
```diff
@@ -95,30 +92,31 @@ Expect the following output:
...
+The namespace in the following commands needs to be replaced with the correct namespace of the previously created service account.
 
 The tasks you can perform depend on your access level.
 
 **Global Admin** can grant broad cluster level admin access by executing the following command:
 
 ```sh
 curl -s https://raw.githubusercontent.com/yugabyte/charts/master/rbac/platform-global-admin.yaml \
-  | sed "s/namespace: <SA_NAMESPACE>/namespace: <namespace>"/g \
+  | sed "s/namespace: <serviceaccount_namespace>/namespace: <namespace>"/g \
```
Comment (yogendra) [inline docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md:105]: `platform-global-admin.yaml` has a placeholder `<SA_NAMESPACE>`. So this updated sed command does not replace the placeholder anymore.

Either
1. Change the command to use the correct placeholder
2. Change the `platform-globle-admin.yaml` to use the correct placeholder

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md [hunk 8]
```diff
@@ -152,11 +150,11 @@ For example, if your goal is to allow the platform software to manage YugabyteDB
 
 ```sh
 curl -s https://raw.githubusercontent.com/yugabyte/charts/master/rbac/platform-namespaced.yaml \
-  | sed "s/namespace: <SA_NAMESPACE>/namespace: <namespace>"/g \
+  | sed "s/namespace: <serviceaccount_namespace>/namespace: <namespace>"/g \
```
Comment (yogendra) [inline docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md:153]: `platform-namespaced.yaml` has a placeholder `<SA_NAMESPACE>`. So this updated sed command does not replace the placeholder anymore.

Either
1. Change the command to use the correct placeholder
2. Change the `platform-namespaced.yaml` to use the correct placeholder

**Review by yogendra — COMMENTED** [review]
Recent changes in the service account setup does not work. The placeholder in the yaml files (<SA_NAMESPACE>) have to be changed to match the new placeholders (<serviceaccount_namespace>).

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md (hunk 5 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md:132]: Changed back to <SA_NAMESPACE>. 
The change will be delivered via a new PR.

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md (hunk 6 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md:113]: Changed back to <SA_NAMESPACE>. 
The change will be delivered via a new PR.

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md (hunk 7 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md:105]: Changed back to <SA_NAMESPACE>. 
The change will be delivered via a new PR.

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md (hunk 8 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md:153]: Changed back to <SA_NAMESPACE>. 
The change will be delivered via a new PR.

**Comment by lizayugabyte** [comment]
Fixed the namespace error in https://github.com/yugabyte/yugabyte-db/pull/11917

**Comment by lizayugabyte** [comment]
@yogendra The error is fixed in https://github.com/yugabyte/yugabyte-db/pull/11917. Could you, please, review and approve

**Pull request merged**